### DISCUSSION TITLE
Fresh install config error?

### DISCUSSION THREAD

**Discussion by xpressivecode** [LazyVim/LazyVim#158] [Q&A] [answered]
I just went through the installation steps on my mac and encountered this:

<img width="1071" alt="image" src="https://user-images.githubusercontent.com/957896/215767455-b307d4e1-7923-4777-ba90-d2c64f70ab02.png">

Any thoughts? This same installation worked fine on my ubuntu desktop. I've tried starting over a few times.

Thanks!

**Comment by folke** [comment, marked as answer]
You should either install using the latest stable NEovim release or a recent nightly. You're currently using an old nightly

**Reply by xpressivecode** [reply]
Sorry for wasting your time... really should have checked that.
